DOL.WHD.Section14c.Domain/Models/Submission/ApplicationSubmission.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace DOL.WHD.Section14c.Domain.Models.Submission
+{
+    public class ApplicationSubmission
+    {
+        public int Id { get; set; }
+
+        #region Application Info
+
+        [Required]
+        public string UserId { get; set; }
+
+        [Required]
+        public virtual Response ApplicationType { get; set; }
+
+        [Required]
+        public bool HasPreviousApplication { get; set; }
+
+        [Required]
+        public bool HasPreviousCertificate { get; set; }
+
+        // TODO: required if HasPreviousCertificate
+        public string CertificateNumber { get; set; }
+
+        [Required]
+        public bool TemporaryAuthority { get; set; }
+
+        [Required]
+        public virtual ICollection<Response> EstablishmentType { get; set; }
+
+        [Required]
+        public string ContactName { get; set; }
+
+        [Required]
+        [Phone]
+        public string ContactPhone { get; set; }
+
+        [Phone]
+        public string ContactFax { get; set; }
+
+        [Required]
+        [EmailAddress]
+        public string ContactEmail { get; set; }
+
+        #endregion
+
+        #region Employer
+
+        [Required]
+        public virtual EmployerInfo Employer { get; set; }
+
+        #endregion
+
+        #region Wage Data
+
+        [Required]
+        public virtual Response PayType { get; set; }
+
+        // TODO: Validation on what WageTypeInfo is based on PayType
+        public virtual WageTypeInfo WageTypeInfo { get; set; }
+
+        #endregion
+
+        #region Work Sites & Employees
+
+        [Required]
+        public int TotalNumWorkSites { get; set; }
```

Comment:
This is sort of unorthodox for us as well.  The form captures this as a field and we plan to validate the actual number of rows against it.  Since it will be constrained with validation, we can probably remove it from the underlying data repository.